File: SwapTwoNumber.c
#include <stdio.h>

void swapTwoNumber(int a, int b){
    
    printf("Số a = %d\nSố b = %d\n", a,b);

    int temp = a; a = b;b = temp;

    printf("\n2 số sau khi hoán đổi:\nSố a = %d\nSố b = %d", a, b);
}

int main(){
    int a; int b;
    printf("Nhập 2 số bất kí cách nhau bởi khoảng trắng: ");
    scanf("\n%d",&a);
    scanf("%d",&b);

    swapTwoNumber(a,b);

    return 0;
}

File: SwapTwoNumber.py
def swapTwoNumber(a,b):
   
    print(f"a = {a}, b = {b} \n")

    print("Hoán đổi 2 số, kết quả là: \n")

    a,b = b,a

    print(f"a = {a}, b = {b}\n")

a, b = map(int, input("Hãy nhập 2 số bất kì cần hoán đổi: ").split())

swapTwoNumber(a,b)
